### PR TITLE
Upgrade to Rails 7.2

### PR DESCRIPTION
**Pull request by jcoyne**
# Why was this change made?

Keep up to date with dependencies 

# How was this change tested?

### CODE REVIEW

File: bin/brakeman [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+require "rubygems"
+require "bundler/setup"
+
+ARGV.unshift("--ensure-latest")
+
+load Gem.bin_path("brakeman", "brakeman")
```
Comment (mjgiarlo) [inline bin/brakeman:7]: Interesting addition. When should we be running `brakeman`? Should it be included in any workflows?

**Review by mjgiarlo — APPROVED** [review]
LGTM. Should we deploy to stage or QA and run a couple integration tests to be safe?

File: bin/setup [hunk 2]
```diff
@@ -30,4 +30,8 @@ FileUtils.chdir APP_ROOT do
 
   puts "\n== Restarting application server =="
   system! "bin/rails restart"
+
+  # puts "\n== Configuring puma-dev =="
+  # system "ln -nfs #{APP_ROOT} ~/.puma-dev/#{APP_NAME}"
+  # system "curl -Is https://#{APP_NAME}.test/up | head -n 1"
```
Comment (aaron-collier) [inline bin/setup:36]: Do we want to maintain this comment as documentation or is it cruft?

Comment (aaron-collier) [reply bin/setup:36]: I see this in all of the other 7.2 PRs, so seems OK to keep.

**Pull request closed without merging**